packages/headless/src/features/facets/range-facets/generic/range-facet-utils.ts

```diff
@@ -3,6 +3,10 @@ import {
   logFacetDeselect,
   logFacetSelect,
 } from '../../facet-set/facet-set-analytics-actions';
+import {
+  logFacetDeselect as logInsightFacetDeselect,
+  logFacetSelect as logInsightFacetSelect,
+} from '../../facet-set/facet-set-analytics-actions';
```

Comment:
Description says 

> The two controllers updated to use the analytics actions imported from the facet-set-insight-analytics-actions file.

But here it's importing the same facet-set-analytics-actions file

Reply:
My bad 😅I just updated the import, thank you!